### ISSUE TITLE
LiveSplit crashes upon startup after updating the software to v. 1.8.29

### ISSUE THREAD

**Issue by StupidP0tato** [LiveSplit/LiveSplit#2507]
Hey

I got a message requesting that I updated LiveSplit so I did, and afterwards I got a crash on startup with the error message you see below: 
![image](https://github.com/LiveSplit/LiveSplit/assets/172418106/7e0a5693-7120-412f-a251-e12c7007d51e)

I tried uninstalling (i.e. deleting the entire folder as I could not find any uninstall.exe file) and downloading the newest version from scratch. Still I get the same message. I tried restarting the PC, nothing. 

I have the following specs:
Windows 11 OS
NVIDIA GeForce RTX 2060
12th Gen Intel Core i9-12900k
64GB RAM

The error log from the PC in the event viewer is as follows: 
![image](https://github.com/LiveSplit/LiveSplit/assets/172418106/80c7c772-9298-4f87-9004-4c21ccf08ebc)

Not sure if relevant, but I also have the following info from the general tab in the event viewer:
![image](https://github.com/LiveSplit/LiveSplit/assets/172418106/5b2b4fae-9074-4622-b12d-4fa5f30c7e15)

Does anyone know what causes this or how I can fix it? 

Best regards
StupidP0tato

**Comment by just-ero** [comment]
Did you extract the .zip archive after re-downloading LiveSplit?

**Comment by StupidP0tato** [comment]
I downloaded the zip file, extracted it and ran the .exe file. I do not have any other virus program than windows defender.

**Comment by ru-mii** [comment]
> I downloaded the zip file, extracted it and ran the .exe file. I do not have any other virus program than windows defender.

Can you take a picture of your LiveSplit folder capturing all of the files inside and send over?

**Comment by StupidP0tato** [comment]
Hey, and thanks all for the very prompt response to this. I didn't expect that to be honest :) 

Here is a screen shot of all items in the folder. 

![image](https://github.com/LiveSplit/LiveSplit/assets/172418106/1094aa7d-0e2a-4344-9c71-bebc6384ac8c)

**Comment by just-ero** [comment]
Please show the entire Windows explorer. The fact that you extracted the files, where to, etc.

**Comment by wooferzfg** [comment]
What's in the `Resources` folder?

**Comment by wooferzfg** [comment]
> I do not have any other virus program than windows defender.

It's possible that Windows Defender could remove some relevant files

**Comment by ru-mii** [comment]
> Hey, and thanks all for the very prompt response to this. I didn't expect that to be honest :)
> 
> Here is a screen shot of all items in the folder.
> 
> ![image](https://private-user-images.githubusercontent.com/172418106/338751837-1094aa7d-0e2a-4344-9c71-bebc6384ac8c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxNDU5MTYsIm5iZiI6MTcxODE0NTYxNiwicGF0aCI6Ii8xNzI0MTgxMDYvMzM4NzUxODM3LTEwOTRhYTdkLTBlMmEtNDM0NC05YzcxLWJlYmM2Mzg0YWM4Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMVQyMjQwMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMDM1ZWQ4NjI0NWU3YjNhNGE3MjBhMDM3ZTE2NjFlYWMwMmRkZWM3MjZlNDRkM2U3OGFiMjdjMmMyODhmODI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.dcCcCC2TIicXS2WPI2rNV7TA2ZAhnCz2Q4xwnq-eLYI)

You didn't show everything as suggested in my response :(

**Comment by StupidP0tato** [comment]
Oops. My bad. I'm posting inside the two folders here, as well as the location: 

Inside Components
![image](https://github.com/LiveSplit/LiveSplit/assets/172418106/14521ada-49e6-472b-8297-445857316baf)


Inside Resources
![image](https://github.com/LiveSplit/LiveSplit/assets/172418106/7b2f3d09-ec77-46d1-a197-14c0342cdc13)

**Comment by wooferzfg** [comment]
This looks like the same issue: https://stackoverflow.com/q/9056418

Not sure what the cause is, though

**Comment by wooferzfg** [comment]
Did you already try running LiveSplit as Admin as suggested above?

**Comment by StupidP0tato** [comment]
> Did you already try running LiveSplit as Admin as suggested above?

Yes. And I have uninstalled it once and reinstalled and I get the same issue.

**Comment by wooferzfg** [comment]
Are there any other event logs at the same time? "Exception has been thrown by the target of an invocation." is referring to some other exception that caused this one.

**Comment by wooferzfg** [comment]
From what I can tell, it's failing to load the LiveSplit icon file, which is [embedded in the `LiveSplit.View.dll` resources](https://github.com/LiveSplit/LiveSplit/tree/master/src/LiveSplit.View#L136). Not sure why this is happening - all the StackOverflow posts about this error mention Windows XP. Maybe some kind of weird issue with the configuration of your computer.

**Comment by wooferzfg** [comment]
This could maybe be related to updating the `Icon.ico` files in https://github.com/LiveSplit/LiveSplit/pull/2442 and https://github.com/LiveSplit/LiveSplit/commit/308b8d280607aec898b421f8e7390a76d65aca18?
@CryZe Have you seen anything like this before?
